### PR TITLE
Add documentation about declaring multiple clients

### LINKED ISSUE #908: [Symfony] Multiple AWS clients
The Symfony bundle allows me to configure my AWS clients like this:

```YAML
async_aws:
    clients:
        s3:
            config:
                # insert S3 configuration here
```

This format assumes that I use the same AWS credentials for all S3 buckets I want to access. However, in my application, this is not the case. I need to access two different S3 buckets in different regions with different credentials. Right now, I need to configure one of the clients manually.

It would be awesome, if the bundle would allow me to configure multiple AWS clients.

### PR DESCRIPTION
**Pull request by jderusse**
Fixes #908 ping @derrabus

### CODE REVIEW

**Review by derrabus — APPROVED** [review]

File: docs/integration/symfony-bundle.md [hunk 1]
```diff
@@ -43,15 +43,17 @@ The config above will create the following services:
 | `async_aws.client.ses`     | AsyncAws\Ses\SesClient
 | `async_aws.client.sqs`     | AsyncAws\Sqs\SqsClient
 | `async_aws.client.delayed` | AsyncAws\Sqs\SqsClient $delayed
-| `async_aws.client.s3`      | AsyncAws\S3\S3Client
-| `async_aws.client.sts`     | AsyncAws\Core\Sts\StsClient
```
Comment (Nyholm) [inline docs/integration/symfony-bundle.md:47]: These should not be removed. 

> This example assume you have installed core, ses, sqs and s3

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you

File: docs/integration/symfony-bundle.md (hunk 1 above)
Comment (jderusse) [reply docs/integration/symfony-bundle.md]: Oh right. I forgot about this behavior.

**Review by Nyholm — APPROVED** [review]

**Comment by Nyholm** [comment]
Thank you

**Pull request merged**